README.md

```diff
@@ -295,7 +300,7 @@ Installation
 ------------
 ```bash
 # Tested on Ubuntu 14.04, for other OSs check out https://github.com/vysheng/tg#installation
-sudo apt-get install libreadline-dev libconfig-dev libssl-dev lua5.2 liblua5.2-dev libevent-dev make unzip git redis-server g++ libjansson-dev libpython-dev
+sudo apt-get install libreadline-dev libconfig-dev libssl-dev lua5.2 liblua5.2-dev libevent-dev make unzip git redis-server g++ libjansson-dev libpython-dev expat libexpat1-dev
```

Comment:
When I installed feedparser from luarocks in my raspberry, in the dependency of feedparser, that is `luaexpat`, it failed to install, and searching they said that I needed that two libraries, so I installed and then feedparser intalled ok from luarocks.
Maybe you already have then installed, or they are no longer needed